R/use_extendr.R

```diff
@@ -68,6 +68,22 @@ use_extendr <- function(path = ".",
     cli::cli_alert_success("Creating {.file {pretty_rel_path(rust_src_dir, path)}}.")
   }
 
+  use_rextendr_template(
+    "configure",
+    save_as = file.path("configure"),
+    quiet = quiet,
+    as_executable = TRUE,
+    overwrite = overwrite
+  )
+
+  use_rextendr_template(
+    "configure.win",
+    save_as = file.path("configure.win"),
+    quiet = quiet,
+    as_executable = TRUE,
+    overwrite = overwrite
+  )
+
```

Comment:
I wonder if it would make more sense to have the Sys.chmod() calls separated out for these or included in the `use_rextendr_template()`. I think this current change is actually quite nice because who knows what will need to be executable in the future.